spring-cloud-stream-module-launcher/src/main/java/org/springframework/cloud/stream/module/launcher/ModuleLauncherRunner.java

```diff
@@ -45,15 +48,16 @@
 	private ModuleLauncher moduleLauncher;
 
 	@Override
-	public void run(ApplicationArguments applicationArguments) throws Exception {
-		String[] launchedModules = moduleLauncherProperties.getModules();
+	public void run(String... unused) throws Exception {
+		List<ModuleWithArguments> modulesToRun = moduleLauncherProperties.modulesWithArguments();
 		if (log.isInfoEnabled()) {
-			log.info("Launching: "
-					+ StringUtils.arrayToCommaDelimitedString(launchedModules)
-					+ " with arguments: "
-					+ StringUtils.arrayToCommaDelimitedString(applicationArguments
-							.getSourceArgs()));
+			StringBuilder sb = new StringBuilder(modulesToRun.size() * 100);
```

Comment:
We can leave the argument out - it improves performance a little bit, but it makes the code harder to read.